pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/barnepensjon/BarnepensjonInnvilgelseFraser.kt

```diff
@@ -115,20 +115,24 @@ object BarnepensjonInnvilgelseFraser {
                     Language.Bokmal to "Du finner mer informasjon på ".expr() + Constants.BARNEPENSJON_URL + ". " +
                             "Hvis du ikke finner svar på spørsmålet ditt, kan du ringe oss på telefon " + kontaktTelefon + " " +
                             "hverdager 9-15. ",
-                    Language.Nynorsk to "".expr(),
-                    Language.English to "".expr()
+                    Language.Nynorsk to "Du finn meir informasjon på ".expr() + Constants.BARNEPENSJON_URL + ". " +
+                            "Dersom du ikkje finn svar på spørsmålet ditt der, kan du ringje oss på telefon " + kontaktTelefon + " " +
+                            ", kvardagar 9–15.",
+                    Language.English to "For more information, visit us online: ".expr() + Constants.BARNEPENSJON_URL + ". " +
```

Comment:
`Constants.Engelse.BARNEPENSJON_URL`?

Reply:
Godt du sier ifra, hadde ikke fått med meg at det var ene constants for engelsk